helm/ako/values.yaml

```diff
@@ -16,7 +16,7 @@ AKOSettings:
   deleteConfig: "false" # Has to be set to true in configmap if user wants to delete AKO created objects from AVI 
   disableStaticRouteSync: "false" # If the POD networks are reachable from the Avi SE, set this knob to true.
   clusterName: "my-cluster" # A unique identifier for the kubernetes cluster, that helps distinguish the objects for this cluster in the avi controller. // MUST-EDIT
-  cniPlugin: "" # Set the string if your CNI is calico or openshift. enum: calico|canal|flannel|openshift 
+  cniPlugin: "" # Set the string if your CNI is calico or openshift. enum: calico|canal|flannel|openshift|antrea
```

Comment:
@sudswasavi : I think we need to do same changes at ako-operator side yaml file as well.

Reply:
Also comment for L7Settings:serviceType  should include "NodePortLocal"

Reply:
Thanks. Added the NodePortLocal change. For AKO-operator would, address in a separate PR.